lib/internal/process/next_tick.js

```diff
@@ -179,6 +140,8 @@ function setupNextTick() {
                  triggerAsyncId,
                  this);
       }
+
+      nextTickQueue.push(this);
```

Comment:
Is there a reason to move this into the constructor? I find moving side-effects into constructors to be surprising about 99 % of the time…

Reply:
Probably not. Was trying to consolidate between the two `nextTick` functions but you're right, this likely doesn't belong here.